src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -43,32 +50,62 @@ internal class FunctionDispatcher : IFunctionDispatcher
         {
             _metricsLogger = metricsLogger;
             _scriptOptions = scriptHostOptions.Value;
+            _environment = environment;
             _languageWorkerChannelManager = languageWorkerChannelManager;
             _eventManager = eventManager;
             _workerConfigs = languageWorkerOptions.Value.WorkerConfigs;
-            _logger = loggerFactory.CreateLogger(ScriptConstants.LogCategoryFunctionDispatcher);
+            _managedDependencyOptions = managedDependencyOptions;
+            _logger = loggerFactory.CreateLogger<FunctionDispatcher>();
+            _workerRuntime = _environment.GetEnvironmentVariable(LanguageWorkerConstants.FunctionWorkerRuntimeSettingName);
+
+            var processCount = _environment.GetEnvironmentVariable(LanguageWorkerConstants.FunctionsWorkerProcessCountSettingName);
+            _maxProcessCount = (processCount != null && int.Parse(processCount) > 1) ? int.Parse(processCount) : 1;
```

Comment:
Do we impose an upper limit here?

Reply:
Good idea to impose upper limit. Will fix it.